### PR TITLE
Kubelet code move: volume / util

### PR DESCRIPTION
**Pull request by pmorie**
Addresses some odds and ends that I apparently missed earlier.  Preparation for kubelet code-move ENDGAME.

cc @kubernetes/sig-node

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/31157)

### CODE REVIEW

File: pkg/kubelet/util.go [hunk 1]
```diff
@@ -128,3 +129,32 @@ func dirExists(path string) bool {
 	}
 	return s.IsDir()
 }
+
+// empty is a placeholder type used to implement a set
+type empty struct{}
+
+// podsByCreationTime makes an array of pods sortable by their creation
+// timestamps.
+// TODO: move into util package
```
Comment (yujuhong) [inline pkg/kubelet/util.go]: How about actually moving this to `kubelet/util/sort`?

Comment (pmorie) [inline pkg/kubelet/util.go]: I'm cool with that.

**Comment by pmorie** [comment]
@yujuhong PTAL

**Comment by tmrts** [comment]
/subscribe I'm interested in sending a couple of refactor PRs once this is merged

**Comment by pmorie** [comment]
@tmrts i have one more planned after this, i'll tag you in

**Comment by yujuhong** [comment]
LGTM. Thanks!

**Comment by tmrts** [comment]
@pmorie thanks SGTM :+1:

**Pull request merged**